strawberryfields/utils/utils.py

```diff
@@ -56,10 +56,28 @@
 import scipy as sp
 from scipy.special import factorial as fac
 
-from .engine import LocalEngine
-from .program_utils import Command
-from .ops import Gate, Channel, Ket
-
+from strawberryfields.engine import LocalEngine
+from strawberryfields.program_utils import Command
+from strawberryfields.ops import Gate, Channel, Ket
+
+__all__ = [
+    "squeezed_cov",
+    "vacuum_state",
+    "coherent_state",
+    "squeezed_state",
+    "displaced_squeezed_state",
+    "fock_state",
+    "cat_state",
+    "randnc",
+    "random_covariance",
+    "random_symplectic",
+    "random_interferometer",
+    "operation",
+    "is_unitary",
+    "is_channel",
+    "extract_unitary",
+    "extract_channel",
+]
```

Comment:
Now that `utils` is a package, we can start to split these into separate modules 🙂 

Groupings that would make sense include:
* `strawberryfields.utils.states`:

  - `squeezed_cov`
  - `vacuum_state`
  - `coherent_state`
  - `squeezed_state`
  - `displaced_squeezed_state`
  - `fock_state`
  - `cat_state`

* `strawberryfields.utils.random`:

  - `random_covariance`
  - `random_symplectic`
  - `random_interferometer`

The rest.. not really sure yet lol